travel/src/main/java/com/tx/travel/security/jwt/JwtUtils.java

```diff
@@ -40,7 +42,15 @@ public String getJwtFromCookies(HttpServletRequest request) {
   }
 
   public ResponseCookie generateJwtCookie(UserDetailsImpl userPrincipal) {
-    String jwt = generateTokenFromUsername(userPrincipal.getUsername());
+    String jwt = generateTokenFromUser(userPrincipal);
+    return ResponseCookie.from(jwtCookie, jwt)
+        .path("/api")
+        .maxAge(24 * 60 * 60L)
```

Comment:
can we make this configurable?

Reply:
Can you be more precise? Do you mean adding more properties to response cookie, as '.secure()' for example or changing path to be less general "./api/..."

Reply:
the comment is referring to line .maxAge,so I mean this number should be part of application, properties and injected here as a value. But now that you mentioned it, "/api" should be a constant at least, if not also part of configuration. Always avoid using hard coded values!